db/types.c

```diff
@@ -9935,9 +9916,7 @@ static void decimal64_ondisk_to_double(server_decimal64_t *pdec64,
                                        decDouble *dn)
 {
     char decimals[DECDOUBLE_PACKED_COEF];
-    int exponent;
-    int sign;
-    int rc = 0;
+    int exponent = 0;
```

Comment:
Looks like exponent always gets a value, why would compiler complain about this one?

Reply:
I dont think in this case compiler is that smart:
/bb/comdb2a/adi/master_branch/db/types.c:9578:15: warning: ‘exponent’ may be used uninitialized in this function [-Wmaybe-uninitialized]
         logmsg(LOGMSG_USER, "Dec64 unmake_order IN (exponent %d %x):\n", *exponent,